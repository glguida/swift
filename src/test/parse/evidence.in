obs Color(b[0]) = Red;