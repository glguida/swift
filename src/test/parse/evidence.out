(BlogProgram:
  (args:
    (Evidence:
      :left
        (FuncApp:
          :func Color
          (args:
            (OpExpr:
              :op 16
              :left
                (VarRef: b )
              :right
                (IntLiteral: 0 )
            )
          )
        )
      :right
        (VarRef: Red )
    )
  )
)
